File: hackerrank/day2/main.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function solve(comida, propina, impuestos) {
    let total = comida + (comida * propina / 100) + (comida * impuestos / 100);

    console.log(Math.round(total));
    console.log(`El total de la comida es: ${Math.round(total)}`);
}

rl.question("Digite el monto de la comida (Decimal): ", (meal) => {
    rl.question("Digite el porcentaje de propina (Entero): ", (tip) => {
        rl.question("Digite el porcentaje de impuestos (Entero): ", (tax) => {

            solve(parseFloat(meal), parseInt(tip), parseInt(tax));

            rl.close();
        });
    });
});

File: hackerrank/day2/main.cs
using System;

class Program
{
    static void Solve(double comida, int propina, int impuestos)
    {
        double total = comida + (comida * propina / 100.0) + (comida * impuestos / 100.0);

        Console.WriteLine(Math.Round(total));
        Console.WriteLine($"El total de la comida es: {Math.Round(total)}");
    }

    static void Main(string[] args)
    {
        Console.Write("Digite el monto de la comida (Decimal): ");
        double mealCost = Convert.ToDouble(Console.ReadLine());

        Console.Write("Digite el porcentaje de propina (Entero): ");
        int tipPercent = Convert.ToInt32(Console.ReadLine());

        Console.Write("Digite el porcentaje de impuestos (Entero): ");
        int taxPercent = Convert.ToInt32(Console.ReadLine());

        Solve(mealCost, tipPercent, taxPercent);
    }
}